### ISSUE TITLE
Initial Option Doesn't Work on Radio Buttons

### ISSUE THREAD

**Issue by johnbhiggins7** [FlutterFlow/flutterflow-issues#196]
## Expected behavior (required)
When "Initial Option" is set the value of Widget State should be equal to the Initial Option.

## Current behavior (required)
The value is null (even thought the radio button will be visually selected).

## To Reproduce (required)
1. Add Radio Button Widget
2. Provide Options
3. Set Initial Value
4. Run project and inspect widget value

## Context (required)
Trying to filter list based on radio button. I can do this but only after I select an option.

## Your environment
* Version of FlutterFlow used: 3.3.4
* Platform (e.g. Web, MacOS Desktop): Web
* Browser name and version: Chrome 107
* Operating system and version (desktop or mobile): Desktop

This has been broken for awhile and brought up many times by the community:
https://community.flutterflow.io/c/discuss-and-get-help/radio-button-initial-option-broken
https://community.flutterflow.io/c/bugs/radio-button-initial-value-from-variable-doesn-t-work
https://community.flutterflow.io/c/bugs/radio-button-initial-option-set-from-variable-does-not-work
https://community.flutterflow.io/c/discuss-and-get-help/unable-to-set-initial-value-of-variable-for-radio-buttons

**Comment by katherineqian** [comment]
Hi @johnbhiggins7, 

Thanks for your report! Closing this as a duplicate of https://github.com/FlutterFlow/flutterflow-issues/issues/2.

**Comment by ShobhitGalaxy2021** [comment]
Still initial option (In radio button )not working when we use custom function or local state it initial value
